docs/guides/VTEX-Platform-Overview/store-architecture/store-architecture.md

```diff
@@ -0,0 +1,68 @@
+
+## Business-to-customer (B2C)
+
+The business-to-customer (B2C) is the most common business model in VTEX and the base for the other business models.
+
+![image](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/docs/guides/VTEX-Platform-Overview/store-architecture/basic-b2c.png)
+
+## Business-to-business (B2B)
+
+The main characteristic of a [business-to-business (B2B)](https://help.vtex.com/en/tutorial/b2b-overview--5vb9SNXhX2bZnkpAh7ADdC) model is that transactions occur between two or more legal entities.
+
+B2B projects are usually more extensive and complex than projects in the B2C business model, as they often feature complex pricing structures, tailored product catalogs, customized user permissions, and extensive integrations with back office systems to manage various business operations effectively
+
+Below, you can see a reference architecture of a B2B store using [B2B Suite](https://developers.vtex.com/docs/apps/vtex.b2b-suite).
+
+![image](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/docs/guides/VTEX-Platform-Overview/store-architecture/basic-b2b.png)
+
+## Franchise accounts (Omnichannel)
+
+This is a very common architecture model that integrates physical stores with ecommerce as [franchise accounts](https://help.vtex.com/en/tutorial/what-is-a-franchise-account--kWQC6RkFSCUFGgY5gSjdl) in VTEX.
+
+![image](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/docs/guides/VTEX-Platform-Overview/store-architecture/franchises-omni.png)
+
+## Multi-language and Multi-currency
+
+Some of our customers are present in multiple countries and want to have every local store running on VTEX. To serve this customer, we provide the multi-language and multi-currency architectures.
+
+### Single account, multi-bindings
+
+In this architecture, there is a single VTEX account bound to multiple languages and currencies, in a [multidomain](https://help.vtex.com/en/tracks/vtex-store-overview--eSDNk26pdvemF3XKM0nK9/4yPqZQyj0t675QpcG7H6yl#multistore) perspective.
+
+![image](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/docs/guides/VTEX-Platform-Overview/store-architecture/multi-currency-language-single-account.png)
+
+### Multi-account
+
+In this architecture, there is a main VTEX account that will act as a seller in all secondary VTEX accounts, acting as [marketplaces](https://help.vtex.com/en/tutorial/estrategias-de-marketplace-na-vtex--tutorials_402#being-a-vtex-marketplace). These secondary accounts are conceived as [new environments](https://help.vtex.com/en/tracks/vtex-store-overview--eSDNk26pdvemF3XKM0nK9/4yPqZQyj0t675QpcG7H6yl#additional-environment).
+
+![image](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/docs/guides/VTEX-Platform-Overview/store-architecture/multi-currency-language-multi-account.png)
+
+>ℹ️ To deep dive into these store architectures, check out the [Store architecture use cases](https://developers.vtex.com/docs/guides/store-architecture-use-cases) guide.
```

Comment:
Reading this part now, I feel like this guide does not add much when compared to https://github.com/vtexdocs/dev-portal-content/pull/1171 _Store architecture use cases_.

- It feels repetitive to divide sections in the same way in both articles.
- I don't think _Store architecture use cases_ would get way too long if we just added a bit more information to the introduction.
- _Store architecture use cases_ seems stronger and more useful in terms of content and I don't think it should be "hidden" under this subcategory cover.

My suggestion is merging this introduction with the use cases guide, so for me the next steps would be:

1. Adjust content from lines 13 to 28 as suggested and then incorporate this part into the _Store architecture use cases_ guide introduction in https://github.com/vtexdocs/dev-portal-content/pull/1171. Make sure all images used in that guide are in that PR as well.
2. Close this PR.
3. Rename the _Store architecture use cases_ guide to _Store architecture_ and use it as the subcategory cover.